resources/assets/js/modules/format-data.js

```diff
@@ -14,10 +14,15 @@ export default function formatData(response, categoryList) {
     };
   });
 
+  let image = "";
+  if (response._embedded["wp:featuredmedia"] !== undefined) {
+    image = response._embedded["wp:featuredmedia"][0].source_url;
+  }
+
   return {
     title: response.title.rendered,
     url: response.link,
-    image: response._embedded["wp:featuredmedia"][0].source_url,
+    image: image,
```

Comment:
なんにも問題ないので変える必要ないんですが、 key と valueにあたる変数 が同じ名前のとき、

```suggestion
    image,
```

こんな風にもかけたりします！

参考
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions

Reply:
知りませんでした…！ありがとうございますm(_ _)m